features/cli/secrets.feature

```diff
@@ -4,10 +4,10 @@ Feature: secrets related scenarios
   # @case_id OCP-10725
   Scenario: deployment hook volume inheritance --with secret volume
     Given I have a project
-    When I run the :secrets client command with:
-      | action | new        |
-      | name   | my-secret  |
-      | source | /etc/hosts |
+    And I run the :create_secret client command with:
+      | secret_type | generic    |   
+      | name        | my-secret  |   
```

Comment:
White spaces at end of line.
Also, could you confirm it works for all 4.x versions ? Generally we can test 4.1 and latest version, which is 4.5